### DISCUSSION TITLE
Champions Meeting Notes 2023-05-05

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#388] [General]
- we forgot to record sorry guys, but here is the draft https://hackmd.io/@mrmetaverse/OMI-Working-Agreement
- Also we are blocked on Open Collective switch. I can't find the notes explaining how to switch
- Last we agreed to switch the website to a year-long-billing

**Comment by fire** [comment]
[Draft OMI Working Agreement and Standards Draft.txt](https://github.com/omigroup/omigroup/files/11409718/Draft.OMI.Working.Agreement.and.Standards.Draft.txt)
